validation/src/main/scala/hmda/validation/rules/lar/validity/V275.scala

```diff
@@ -0,0 +1,23 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Result }
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+import scala.util.Try
+
+object V275 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val actionTakenDate = lar.actionTakenDate
+    val dateReceived = lar.loan.applicationDate
+
+    when(dateReceived not equalTo("NA")) {
+      Try((actionTakenDate - dateReceived.toInt) is greaterThanOrEqual(0))
```

Comment:
`lar.loan.applicationDate` can be either "NA" or a date, so it is stored as a `String` object.

`lar.actionTakenDate` always has to be a date, so it is stored as an `Int` object.